RIS.js

```diff
@@ -1266,6 +1268,22 @@ function processTag(item, tagValue, risEntry) {
 				zField = ['extra'];
 			}
 		break;
+		case "H1":
+			//H1, H2 can have multiple occurences which cannot be saved
+			if(item.libraryCatalog) {
+				item.multipleLibraries = value;
+				value = '';
+			}
+		break;
+		case "H2":
+			//H1, H2 can have multiple occurences which cannot be saved
+			if(item.callNumber) {
+				zField = ['unsupported', 'H1/H2'];
```

Comment:
The label can be more informative than "H1/H2", which doesn't mean much to the user.